### ISSUE TITLE
Figure out a way to use the `fuels` package in our playwright tests

### ISSUE THREAD

**Issue by Dhaiwat10** [FuelLabs/fuels-ts#2494]
It would be helpful to be able to use the `fuels` package as a dependency in our playwright tests for things like asserting balances etc.

**Comment by arboleya** [comment]
Using the internal `fuels` package when asserting tests can be tricky because the same version would affect the _tests_ **and** the _assertions_ - unless we used an older version from NPM. 

Maybe we could use static ranges when asserting numbers in the UI?

**Comment by Dhaiwat10** [comment]
Hmm for now what we could do is just check the balance of the burner wallet using the number printed in the UI in the top right. That would work but is a bit gimmicky.

**Comment by arboleya** [comment]
Not necessarily. That's how most UI assertions are performed, anyway. First, you render something and then add assertions using elements from the DOM. The benefit is that we'll always validate what the user is actually seeing. 🙂
